store/mockstore/unistore/cophandler/mpp.go

```diff
@@ -44,8 +44,7 @@ const (
 )
 
 const (
-	taskInit int32 = iota
-	taskRunning
+	taskRunning int32 = iota
```

Comment:
```suggestion
	taskRunning int32 = iota +1
```
Maybe using the above code make this logic more secure.